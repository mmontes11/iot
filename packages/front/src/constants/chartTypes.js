export const BARCHART = "BARCHART";
export const LINECHART = "LINECHART";
export const REALTIME = "REALTIME";
