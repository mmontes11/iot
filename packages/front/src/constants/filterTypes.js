export const THING_FILTER_TYPE = "thing";
export const DATE_FILTER_TYPE = "date";
export const FILTER_TYPES = [THING_FILTER_TYPE, DATE_FILTER_TYPE];
