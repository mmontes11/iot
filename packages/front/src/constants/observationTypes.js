export const EVENT_TYPE = "event";
export const MEASUREMENT_TYPE = "measurement";
export const OBSERVATION_TYPES = [EVENT_TYPE, MEASUREMENT_TYPE];
