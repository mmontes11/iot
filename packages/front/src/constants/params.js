export const TYPE = "TYPE";
export const OBSERVATION = "OBSERVATION";
export const GROUPBY = "GROUPBY";
export const THING = "THING";
