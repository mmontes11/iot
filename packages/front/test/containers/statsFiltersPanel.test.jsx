import React from "react";
import { shallow } from "enzyme";
import { Provider } from "react-redux";
import { MemoryRouter } from "react-router-dom";
import FiltersPanelContainer from "containers/filtersPanelContainer";
import { defaultStore } from "../constants";

describe("containers/statFiltersPanel", () => {
  it("renders in in initial state", () => {
    const wrapper = shallow(
      <Provider store={defaultStore}>
        <MemoryRouter initialEntries={["/"]} keyLength={0}>
          <FiltersPanelContainer onFiltersChange={() => undefined} />
        </MemoryRouter>
      </Provider>,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
