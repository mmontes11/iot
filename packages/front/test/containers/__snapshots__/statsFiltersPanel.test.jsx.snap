// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers/statFiltersPanel renders in in initial state 1`] = `
<MemoryRouter
  initialEntries={
    Array [
      "/",
    ]
  }
  keyLength={0}
>
  <Connect(FiltersPanelContainer)
    onFiltersChange={[Function]}
  />
</MemoryRouter>
`;
